hugo.toml

```diff
@@ -14,7 +14,7 @@ summaryLength = 10 # see https://gohugo.io/content-management/summaries/
 # google analytics
 googleAnalytics = "G-MEASUREMENT_ID" # see https://gohugo.io/templates/internal/#configure-google-analytics
 # disqus short name
-disqusShortname = "" # we use disqus to show comments in blog posts. To install disqus please follow this tutorial https://portfolio.peter-baumgartner.net/2017/09/10/how-to-install-disqus-on-hugo/
+disqus.shortname = "" # we use disqus to show comments in blog posts. To install disqus please follow this tutorial https://portfolio.peter-baumgartner.net/2017/09/10/how-to-install-disqus-on-hugo/
```

Comment:
Was this a manual change or from a deps update?

Reply:
Had to change it to run the application